validator/client/beacon-api/submit_aggregate_selection_proof.go

```diff
@@ -80,7 +138,7 @@ func (c *beaconApiValidatorClient) aggregateAttestation(
 	params := url.Values{}
 	params.Add("slot", strconv.FormatUint(uint64(slot), 10))
 	params.Add("attestation_data_root", hexutil.Encode(attestationDataRoot))
-	endpoint := buildURL("/eth/v1/validator/aggregate_attestation", params)
+	endpoint := buildURL("/eth/v2/validator/aggregate_attestation", params)
```

Comment:
depends on the missing implementation

Reply:
`/eth/v2/validator/aggregate_attestation` has been merged in so is no longer missing